hello.rb

```diff
@@ -171,6 +204,11 @@ def self.validate(user)
   return erb :error
 end
 
+get '/page/:page' do
```

Comment:
Please remove this, since it doesn't seem to do anything right now, and the erb template is missing.